File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.py
arr = input().split('-')

num = []
for i in arr:
    sum = 0
    temp = i.split('+')
    for j in temp:
        sum += int(j)
    num.append(sum)

answer = num[0]

for n in range(1, len(num)):
    answer -= num[n]

print(answer)

File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "ex.txt";
const input = fs.readFileSync(filePath).toString().trim().split("\n");

const arr = input[0].split("-");
num = [];
for (const a of arr) {
  let sum = 0;
  const temp = a.split("+");
  for (const t of temp) {
    sum += Number(t);
  }
  num.push(sum);
}

ans = num[0];

for (let i = 1; i < num.length; i++) {
  ans -= num[i];
}

console.log(ans);
